include/files.php

```diff
@@ -1,6 +1,6 @@
 <?php
 function create_files_from_item($itemid) {
-	global $a;
+	$a = get_app();
```

Comment:
Yeah, but please don't add any additional things to this pull request. This is a real huge monster that we shouldn't feed which much more stuff.